utils/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/InputMeta.scala

```diff
@@ -44,34 +59,20 @@ object InputMeta {
     // TODO: exclude non-key parameters to trait BaseKafkaInputMetaVariables and use it in TypesInformationExtractor.mandatoryClasses
     new TypedObjectTypingResult(
       ListMap(
-        keyParameterName -> keyTypingResult,
-        "topic"          -> Typed[String],
-        "partition"      -> Typed[Integer],
-        "offset"         -> Typed[java.lang.Long],
-        "timestamp"      -> Typed[java.lang.Long],
-        "timestampType"  -> Typed[TimestampType],
-        "headers"        -> Typed.genericTypeClass(classOf[java.util.Map[_, _]], List(Typed[String], Typed[String])),
-        "leaderEpoch"    -> Typed[Integer]
+        keyParameterName           -> keyTypingResult,
+        topicParameterName         -> Typed[String],
+        partitionParameterName     -> Typed[Integer],
+        offsetParameterName        -> Typed[java.lang.Long],
+        timestampParameterName     -> Typed[java.lang.Long],
+        timestampTypeParameterName -> Typed[TimestampType],
+        headersParameterName -> Typed
+          .genericTypeClass(classOf[java.util.Map[_, _]], List(Typed[String], Typed[String])),
+        leaderEpochParameterName -> Typed[Integer]
       ),
-      Typed.typedClass[InputMeta[AnyRef]]
+      Typed.typedClass[java.util.Map[_, _]]
     ) {
-      override def display: String = Typed.genericTypeClass(classOf[InputMeta[_]], List(keyTypingResult)).display
+      override def display: String = s"InputMeta[${keyTypingResult.display}]"
```

Comment:
Why not just:
```
keyTypingResult.display
```
?

Reply:
Because this is a structure that also holds other information than the key: topic name, timestamp and many others. I didn't change it - effectively it behave the same as before the change. I think that @gskrobisz overrides this `display` method to avoid presentation of verbose type with listed all fields of type. Am I right @gskrobisz ?